src/reducers/_account.js

```diff
@@ -250,6 +250,16 @@ export const accountUpdateAccountAddress = (accountAddress, accountType) => (
   }
 };
 
+export const accountChangeLanguage = language => dispatch => {
+    //TODO: needs to trigger render after change
```

Comment:
Thinking about best way to trigger a re-render. Either: 
- Wrap the main `Router` component with an HOC using react-i18next (see https://github.com/i18next/react-i18next/blob/master/example/react_withHOC/src/App.js)
- Trigger a forceUpdate?
- Another way I haven't thought about but @pedrouid has?

Reply:
You are doing great, it's quite simple actually. Because you already control the language selected on the account reducer, you can simply mapStateToProps on the Router and this will be enough to trigger the change.

On the `Router.js`
```
const reduxProps = ({ account }) => ({
  language: account.language
});

export default withRouter(
  connect(reduxProps, {
    warningOffline,
    warningOnline
  })(Router)
);
```